<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>511</width>
     <height>281</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="commonTab">
    <attribute name="title">
     <string>Common settings</string>
    </attribute>
    <widget class="QPushButton" name="pathbutton">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>230</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLabel" name="namelabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nameinput">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="versionlabel">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>10</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string>Version</string>
     </property>
    </widget>
    <widget class="QComboBox" name="versioninput">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>10</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="currentText">
      <string/>
     </property>
    </widget>
    <widget class="QTextEdit" name="descriptioninput">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>321</width>
       <height>181</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
    </widget>
    <widget class="QLabel" name="descriptionlabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>61</width>
       <height>181</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string>Description</string>
     </property>
    </widget>
    <widget class="QLabel" name="pathview">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>461</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="iconview">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>40</y>
       <width>101</width>
       <height>101</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="chooseicon">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>142</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Icon</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="texturesTab">
    <attribute name="title">
     <string>Textures settings</string>
    </attribute>
    <widget class="QCheckBox" name="c_block">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Blocks</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="c_colormap">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Colormap</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="c_entity">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Entities</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="c_environment">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Environment</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="c_gui">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>GUI</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="c_item">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Items</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="c_map">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>10</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Map icons</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="c_misc">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Misc textures</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="c_mob_effect">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>70</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Mob effects</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="c_painting">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>130</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Paintings</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="c_particle">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>160</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Particles</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="c_font_textures">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Fonts</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="c_armor_models">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>100</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Armor</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="otherTab">
    <attribute name="title">
     <string>Other settings</string>
    </attribute>
    <widget class="QCheckBox" name="c_blockstates">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Blockstates</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="c_models">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Models</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="c_lang">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Language</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="c_fonts">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Fonts</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="c_end">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>End poem</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="c_splashes">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Splashes</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="optifineTab">
    <attribute name="title">
     <string>OptiFine settings</string>
    </attribute>
    <widget class="QCheckBox" name="c_cit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Custom items</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="c_cem">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Custom entities</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="c_ctm">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Connected textures</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="c_anim">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Animations</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="c_gui_o">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>10</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>GUI</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="c_lightmap">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>40</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Lightmaps</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="c_loading">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>70</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Custom loading screen</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="c_panoramas">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>100</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Custom panoramas</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="c_sky">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>130</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Custom skies</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="c_dynamic_light">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>160</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Dynamic lights</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="c_emissive">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Emmisive textures</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="c_random_entities">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>190</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Random Entities</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="c_better_grass">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Better grass</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="c_random_paintings">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Random paintings</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>300</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
